### DISCUSSION TITLE
gnome+hyprland

### DISCUSSION THREAD

**Discussion by Vikash-Singh-Tanwar** [1amSimp1e/dots#26] [Q&A] [answered]
can you please make a script about the installation or make a script to use hyprland in gnome

**Comment by 1amSimp1e** [comment, marked as answer]
gnome is a desktop manager and hyprland is windows manager. That doesn't make sense to use a windows maanger on a desktop manager. You will need to use one GNOME or Hyprland thats all

**Reply by Vikash-Singh-Tanwar** [reply]
sorry to say it but i have an hybrid grphics on my laptop with nvidia and i went through hell and it wont work perfectly unlike i use ubuntu which have driver finding software of some sort and i found a way to use hyprland on debian do i thought if he is using gnomw+hyprland so i thought it might work sorry for any trouble

**Reply by 1amSimp1e** [reply]
its ok
